### ISSUE TITLE
Export-NavContainerObjects failing in the split function

### ISSUE THREAD

**Issue by StephenDoherty** [microsoft/navcontainerhelper#616]
Hello,

I am trying to export the objects from my container but receive the following error when it gets to the split section of the powershell script. The main object get exported but the split fails.

Here is my command: (limited to table 18 for testing)
`Export-NavContainerObjects -containerName NAV2015test -objectsFolder "C:\ProgramData\NavContainerHelper\Extensions\Steve-Originals" -filter "Type=table;ID=18" -exportTo 'txt folder'` 

Here is the output and error:

```
PS H:\> Export-NavContainerObjects -containerName NAV2015test -objectsFolder "C:\ProgramData\NavContainerHelper\Extensions\Steve-Originals" -filter "Type=table;ID=18" -exportTo 'txt folder' 
Export Objects with filter 'Type=table;ID=18' to C:\ProgramData\NavContainerHelper\Extensions\Steve-Originals\objects.txt (container path)
Split C:\ProgramData\NavContainerHelper\Extensions\Steve-Originals\objects.txt to C:\ProgramData\NavContainerHelper\Extensions\Steve-Originals (container paths)
WARNING: UnhandledErrorMessage
The type initializer for 'Microsoft.Dynamics.Nav.Model.TypeSystem.WindowsLanguageHelper' threw an exception.
at <ScriptBlock>, <No file>: line 70
The type initializer for 'Microsoft.Dynamics.Nav.Model.TypeSystem.WindowsLanguageHelper' threw an exception.
At C:\Program Files\WindowsPowerShell\Modules\navcontainerhelper\0.6.4.1\ContainerHandling\Invoke-ScriptInNavContainer.ps1:37 char:13
+             Invoke-Command -Session $session -ScriptBlock $scriptbloc ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Split-NAVApplicationObjectFile], TypeInitializationException
    + FullyQualifiedErrorId : System.TypeInitializationException,Microsoft.Dynamics.Nav.Model.Tools.Cmdlets.SplitNavApplicationObjectFile
    + PSComputerName        : 57da84ce82e8a4804012f2f36cc32edf440fed2e248143f90bb3e9cb8bec0eb7
```

**Comment by freddydk** [comment]
This is likely a variant of this: https://github.com/microsoft/navcontainerhelper/issues/610
Windows Server 2019 containers are not very compatible with non-unicode applications like finsql.
Typically, if you run process isolation (-usebestcontaineros) - or manually select to run the ltsc2016 version - then it should work.

**Comment by freddydk** [comment]
Shipped in 0.6.4.2
